syntax = "proto3";
package protos.playbooks;

import "google/protobuf/wrappers.proto";
import "google/protobuf/struct.proto";


message PagerDuty {
  message SendNote {
    google.protobuf.StringValue service = 1;
    google.protobuf.StringValue incident = 2;
    google.protobuf.Struct note = 3;
  }

  enum TaskType {
    UNKNOWN = 0;
    SEND_NOTE = 1;
  }

  TaskType type = 1;
  oneof task {
    SendNote send_note = 2;
  }
}